packages/alfa-rules/src/sia-er65/diagnostics.ts

```diff
@@ -0,0 +1,60 @@
+import { Diagnostic } from "@siteimprove/alfa-act";
+
+export type Reason = "auto-detected" | "user-answered" | "good-class";
```

Comment:
I think it's better to use an enum for cases like that (which we don't always do 😞) This provides safer typing.
```suggestion
export enum Reason = 
  | AutoDetected = "auto-detected" 
  | UserAnswered = "user-answered" 
  | GoodClass = "good-class";
```